app/_locales/ko/messages.json

```diff
@@ -1039,6 +1258,18 @@
   "shapeshiftBuy": {
     "message": "Shapeshift를 통해서 구매하기"
   },
+  "showAdvancedGasInline": {
+    "message": "고급 가스 제어"
+  },
+  "showAdvancedGasInlineDescription": {
+    "message": "전송 및 확인 화면에서 직접 가스 가격과 한도 제어를 표시하려면 이 옵션을 선택해주세요."
+  },
+  "showFiatConversionInTestnets": {
+    "message": "테스트 넷에서 변환 보여주기"
```

Comment:
This sentence doesn't have a word for `Fiat`

Reply:
👍 I will fix it